frontend/src/components/HostSignupForm.tsx

```diff
@@ -0,0 +1,298 @@
+            </NumberInputStepper>
+          </NumberInput>
+        </FormControl>
+        <FormControl>
+          <CheckboxGroup onChange={(v) => restrictions.field.onChange(v)}>
+            <Box>
+              <FormLabel>
+                <Trans>Group Members</Trans>
+              </FormLabel>
+              <SimpleGrid columns={[2, null, 3]} spacing='10px'>
+                <Checkbox id='men' value='Adult men'>
+                  Adult Men
+                </Checkbox>
+                <Checkbox id='women' value='Adult women'>
+                  Adult Women
+                </Checkbox>
+                <Checkbox id='minors' value='Kids'>
+                  Minors(0-16)
+                </Checkbox>
+              </SimpleGrid>
+            </Box>
+            <Box pt='16px'>
+              <FormLabel>
+                <Trans>Pets</Trans>
+              </FormLabel>
+              <SimpleGrid columns={[2, null, 3]} spacing='10px'>
+                <Checkbox id='dogs' value='Dogs'>
+                  Pet Dog
+                </Checkbox>
+                <Checkbox id='cats' value='Cats'>
+                  Pet Cat
+                </Checkbox>
+                <Checkbox id='pets' value='Other pets'>
+                  Other Pet
+                </Checkbox>
+              </SimpleGrid>
+            </Box>
+          </CheckboxGroup>
+        </FormControl>
+        <FormControl>
+          <FormLabel htmlFor='shelter'>
+            <Trans>Shelter</Trans>
+          </FormLabel>
+          <Textarea id='shelter' {...register('summary.shelter')} />
+        </FormControl>
+        <Divider />
+        <FormControl display='flex' alignItems='center'>
+          <FormLabel htmlFor='avaliable' mb='0'>
+            <Trans>Avaliable</Trans>
+          </FormLabel>
+          <Switch
+            id='avaliable'
+            onChange={(state) => active.field.onChange(state)}
+          />
+        </FormControl>
+        <FormControl>
+          <FormLabel htmlFor='avaliability-date'>
+            <Trans>Availability</Trans>
+          </FormLabel>
+          <DatePicker
+            id='avaliability-date'
+            selected={parseDate(dateAvailable.field.value)}
+            onChange={(date) =>
+              dateAvailable.field.onChange(formatDate(date as Date))
+            }
+            disabled={!getValues('summary.availability.active')}
+            showPopperArrow={true}
+            dateFormat='dd-MM-yyyy'
+          />
+        </FormControl>
+        <FormControl>
+          <FormLabel htmlFor='region'>
+            <Trans>Length of stay</Trans>
+          </FormLabel>
+          <Select
+            id='region'
+            {...register('summary.availability.length_of_stay')}
+          >
+            <option key='Less than 3 days' value='Less than 3 days'>
+              {i18n._('less_3_days')}
+            </option>
+            <option key='One week or less' value='One week or less'>
+              {i18n._('one_week_less')}
```

Comment:
Here I think you just do:
```jsx
<option ...> 
   <Trans>One week or less</Trans>
</option>
```

Reply:
It doesnt work, it shows `[object Object]`, it's a known issue. So, somewhere I've read we need to use like this ` {i18n._('one_week_less')}` but as far I understood, all key translations is auto-generates, we cant add it manually?